openml/tasks/task.py

```diff
@@ -145,9 +145,9 @@ def _get_repr_body_fields(self) -> Sequence[tuple[str, str | int | list[str]]]:
         ]
         return [(key, fields[key]) for key in order if key in fields]
 
-    def get_dataset(self) -> datasets.OpenMLDataset:
+    def get_dataset(self, **kwargs) -> datasets.OpenMLDataset:
```

Comment:
Please add a note to the documentation that it accepts arguments for the `datasets.get_dataset` call so the caller knows what kwargs are allowed.

Reply:
Sure, will do in a few hours. Thanks for taking a look!